diff --git a/Makefile b/Makefile
index ac62468..d9bcf86 100644
--- a/Makefile
+++ b/Makefile
@@ -79,7 +79,7 @@ CFLAGS := -O2 -fno-strict-aliasing -fomit-frame-pointer \
 		  -Wall -pipe -g -fwrapv -arch i386 -arch x86_64
 else
 CFLAGS := -O2 -fno-strict-aliasing -fomit-frame-pointer \
-		  -Wall -pipe -g -MMD -fwrapv -fPIC -msse2 -mfpmath=sse -Wno-unused-result
+		  -Wall -pipe -g -MMD -fwrapv -fPIC -msse2 -mfpmath=sse -Wno-unused-result -Wno-unknown-pragmas
 endif
 
 # ----------
diff --git a/src/bot/func.c b/src/bot/func.c
index 22b8d6c..0beb853 100644
--- a/src/bot/func.c
+++ b/src/bot/func.c
@@ -86,8 +86,7 @@ void Load_BotInfo()
 	gamepath = gi.cvar ("game", "0", CVAR_NOSET);
 
 	//load info
-	//sprintf(Buff,"%s/3ZBConfig.cfg",gamepath->string);
-	sprintf(Buff,"%s/%s/3zbconfig.cfg",GET_BASEPATH_STR(),gamepath->string);
+	sprintf(Buff,"%s\\%s\\3zbconfig.cfg",GET_BASEPATH_STR(),gamepath->string);
 	fp = fopen(Buff,"rt");
 	if(fp == NULL)
 	{
diff --git a/src/g_ctf.c b/src/g_ctf.c
index 7da1da0..b7583d9 100644
--- a/src/g_ctf.c
+++ b/src/g_ctf.c
@@ -3017,11 +3017,8 @@ void CTFSetupNavSpawn()
 
 	if(ctf->value) gi.dprintf("We are in CTF mode.\n");
 
-	if(!ctf->value) sprintf(name,"%s/%s/chdtm/%s.chn",GET_BASEPATH_STR(),gamepath->string,level.mapname);
-	else sprintf(name,"%s/%s/chctf/%s.chf",GET_BASEPATH_STR(),gamepath->string,level.mapname);
-
-	//if(!ctf->value) sprintf(name,"%s/chdtm/%s.chn",gamepath->string,level.mapname);
-	//else sprintf(name,"%s/chctf/%s.chf",gamepath->string,level.mapname);
+	if(!ctf->value) sprintf(name,"%s\\%s\\chdtm\\%s.chn",GET_BASEPATH_STR(),gamepath->string,level.mapname);
+	else sprintf(name,"%s\\%s\\chctf\\%s.chf",GET_BASEPATH_STR(),gamepath->string,level.mapname);
 
 	fpout = fopen(name,"rb");
 	if(fpout == NULL)
diff --git a/src/g_main.c b/src/g_main.c
index e5b32f7..d1bea09 100644
--- a/src/g_main.c
+++ b/src/g_main.c
@@ -225,8 +225,8 @@ void Get_NextMap()
 	
 	if(!maplist->string) return;
 
-	//sprintf(Buff,".\\%s\\3ZBMAPS.LST",gamepath->string);
-	sprintf(Buff,"%s/%s/3zbmaps.lst",GET_BASEPATH_STR(),gamepath->string);
+	sprintf(Buff,"%s\\%s\\3zbmaps.lst",GET_BASEPATH_STR(),gamepath->string);
+
 	fp = fopen(Buff,"r");
 	if(fp == NULL) return;
 	
diff --git a/src/g_svcmds.c b/src/g_svcmds.c
index c694cd5..7955ffc 100644
--- a/src/g_svcmds.c
+++ b/src/g_svcmds.c
@@ -312,12 +312,8 @@ void SaveChain()
 		return;
 	}
 
-	//とりあえずCTFだめ
-	//if(ctf->value) 	sprintf(name,".\\%s\\chctf\\%s.chf",gamepath->string,level.mapname);
-	//else 	sprintf(name,".\\%s\\chdtm\\%s.chn",gamepath->string,level.mapname);
-
-	if(ctf->value) 	sprintf(name,"%s/%s/chctf/%s.chf",GET_BASEPATH_STR(),gamepath->string,level.mapname);
-	else 	sprintf(name,"%s/%s/chdtm/%s.chn",GET_BASEPATH_STR(),gamepath->string,level.mapname);
+	if(ctf->value) 	sprintf(name,"%s\\%s\\chctf\\%s.chf",GET_BASEPATH_STR(),gamepath->string,level.mapname);
+	else 	sprintf(name,"%s\\%s\\chdtm\\%s.chn",GET_BASEPATH_STR(),gamepath->string,level.mapname);
 
 	fpout = fopen(name,"wb");
 	if(fpout == NULL) gi.cprintf(NULL,PRINT_HIGH,"Can't open %s\n",name);
diff --git a/src/header/local.h b/src/header/local.h
index 0c711f7..8a70923 100644
--- a/src/header/local.h
+++ b/src/header/local.h
@@ -566,10 +566,10 @@ extern	int	meansOfDeath;
 
 extern	edict_t			*g_edicts;
 
-#define	FOFS(x) (long)&(((edict_t *)0)->x)
-#define	STOFS(x) (long)&(((spawn_temp_t *)0)->x)
-#define	LLOFS(x) (long)&(((level_locals_t *)0)->x)
-#define	CLOFS(x) (long)&(((gclient_t *)0)->x)
+#define	FOFS(x) (int)&(((edict_t *)0)->x)
+#define	STOFS(x) (int)&(((spawn_temp_t *)0)->x)
+#define	LLOFS(x) (int)&(((level_locals_t *)0)->x)
+#define	CLOFS(x) (int)&(((gclient_t *)0)->x)
 
 #define random()	((rand () & 0x7fff) / ((float)0x7fff))
 #define crandom()	(2.0 * (random() - 0.5))
